python/ray/scripts/scripts.py

```diff
@@ -876,10 +870,11 @@ def submit(cluster_config_file, docker, screen, tmux, stop, start,
 @click.argument("cluster_config_file", required=True, type=str)
 @click.argument("cmd", required=True, type=str)
 @click.option(
-    "--docker",
-    is_flag=True,
-    default=False,
-    help="Runs command in the docker container specified in cluster_config.")
+    "--run-env",
+    required=False,
+    type=str,
+    default="auto",
+    help="Chooses running location. Either auto, host, or docker.")
```

Comment:
Can you provide more help text here? A user would ask "what's a running location", and "what does this have to do with my script"? Also, make sure to explain when you might run into failures.